### DISCUSSION TITLE
Is now the time to embrace myst for tutorial notebooks or is it too early?

### DISCUSSION THREAD

**Discussion by fmaussion** [executablebooks/meta#1129] [Q&A] [answered]
We are using jupyter-book as a tool to online-display a bunch of computational / tutorial notebooks for our numerical model: https://oggm.org/tutorials/stable

The purpose of the website is to have a rendered reference for users and learners, and encourage them to download the notebooks and run them locally or on our hub. Notebooks are central, the website is "nice to have".

So far we are not using myst syntax because it does not render well in jupyter-lab per default. But this is annoying and creates "ugliness", because we'd like to use all the features available in myst such as internal references, blocks, etc.

We can enforce our users to install `jupyterlab-myst`, or we can wait for `jupyterlab-myst` to become installed per default in jupyterlab. Does anyone know the timeline on the latter? Will myst simply become the standard syntax in jupyter notebooks soon? I certainly hope so!

**Comment by agoose77** [comment, marked as answer]
Hi @fmaussion, sorry for the late reply on this one: it slipped through the cracks.

RE `jupyterlab-myst`, I suspect it will not become a "default" for a long time, if it does. We're moving fast and iterating on things in this space, whereas CommonMark is somewhat stable (unlike GFM) which is probably one less thing for the JupyterLab maintainers to have to figure out.

**Comment by fmaussion** [comment]
Thanks @agoose77 ! This is very helpful.

Regardless of syntax I really think that (cross-)references between notebooks cells and notebooks would be fantastic additions to jupyter-lab, since notebooks are rarely "standalone documents" anymore. 

I will consider enforcing jupyterlab-myst on users of our tutorials. It's easy to do on binder or hub, but having everyone installing it locally is a bit more of a hassle.
